lib/vagrant-libvirt/config.rb

```diff
@@ -659,7 +659,13 @@ def finalize!
         @management_network_domain = nil if @management_network_domain == UNSET_VALUE
         @system_uri      = 'qemu:///system' if @system_uri == UNSET_VALUE
 
-        @qemu_use_session = false if @qemu_use_session == UNSET_VALUE
+        if ENV['VAGRANT_LIBVIRT_QEMU_USE_SESSION'] == 'true'
```

Comment:
You could write this as:
```
if @qemu_use_session == UNSET_VALUE
  @qemu_use_session = ENV['VAGRANT_LIBVIRT_QEMU_USE_SESSION'] == 'true'
end
```

Reply:
Thanks! Made these changes in https://github.com/vagrant-libvirt/vagrant-libvirt/pull/997/commits/fc62fd27c93ff554a608f52e4d89ee602f07ec22 :man_dancing: